### DISCUSSION TITLE
Location via ASCOM telescope

### DISCUSSION THREAD

**Discussion by mmalziotti** [Stellarium/stellarium#3723] [Ideas]
Good evening
ASCOM provides methods to retrieve site longitude and latitude (see: https://ascom-standards.org/Help/Developer/html/Fields_T_ASCOM_Astrometry_SiteInfo.htm).
If I connect to a telescope via ASCOM from the "Telescope Control" plug-in, the view on the planetarium is NOT updated with the position of the telescope available from the ASCOM connection.
Why? I am forced to reiterate the same position by selecting the appropriate "Location" icon on stellarium.
A radio button with the following choices may be useful:
- manually enter coordinate lat/lon
- choose location from the list
- import coordinate from ASCOM
as for example done by other software (see https://www.astrophotography.app/usersguide/location.htm)
Thank you.
Marco Malziotti

**Comment by gzotti** [comment]
Why? Because nobody has implemented that because it is not a standard feature provided by all telescopes.  Anybody who would expect and be interested to have and use such feature, including you, could add it to this open-source project. But it seems everybody expects somebody else to do it...
